Source/Objects/Misc Objects/InFluxDB/ORInFluxDBModel.m

```diff
@@ -578,68 +578,56 @@ - (void) setErrorString:(NSString*)anError
 
 }
 
+- (void) updateAlarmState:(ORAlarm*)anAlarm
```

Comment:
This change deduplicates the code.
Also changes the schema in a way, which is more consistent and easier to plot.